front/app/hooks/useAdminPublicationsStatusCounts.ts

```diff
@@ -42,13 +43,18 @@ export default function useAdminPublicationsStatusCounts({
     areas.length === 0 ? setAreas(undefined) : setAreas(areas);
   }, []);
 
+  const onChangeSearch = useCallback((search: string) => {
```

Comment:
Not sure why we need useCallback here? Also, does it do anything if you don't pass a dependency?